### DISCUSSION TITLE
Trend Security Agent find a virus when importing pdfplumber

### DISCUSSION THREAD

**Discussion by dremoussenard** [jsvine/pdfplumber#499] [Q&A]
I'm working with python 3.7 and kivy. And when I import pdfplumber in my code, and produce the exe with pyInstaller, I got an alert fromTrend Micro Security Agent (trojan win32 etc). Compiling without the pdfplulmber gives a clean exe.

How to control that everything is ok?

I have made a test with virustotal, and 2 on 66 engines found something. Is it a false true?

Thank you
Didier
![trend](https://user-images.githubusercontent.com/1711028/132213329-d7449776-ab50-4477-aa68-467973401770.PNG)

**Comment by jsvine** [comment]
Hi @dremoussenard, and thank you for the report. I think this may relate to https://github.com/pdfminer/pdfminer.six/issues/363, in which a PDF was the subject of a Windows virus alert. It sounds like it has been fixed in that library, which is a dependency of this project; we are waiting for the next release before upgrading the `pdfminer.six`, however, due to some bugs in the current public version.
